terraform/deployments/govuk-publishing-platform/backends_origin.tf

```diff
@@ -130,6 +131,13 @@ resource "aws_lb_target_group" "publisher" {
   health_check {
     path = "/healthcheck"
   }
+
+  tags = merge(
+    local.additional_tags,
+    {
+      name      = "backend-origin-publisher-tg"
```

Comment:
based on my new #231, it should be just `publisher-tg`

Reply:
i will have a look now and merge your change